docs/UserGuide.md

```diff
@@ -148,15 +143,22 @@ Format: `edit INDEX [id/PATIENT_HOSPITAL_ID] [n/NAME] [p/PREFERRED_NAME] [f/FOOD
 * Edits the patient at the specified `INDEX`. The index refers to the index number shown in the displayed patient list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* * The `edit` command accepts parameters which consists of:
-    *  `patientHospitalId` integer,
-    * `name`, `preferredName` String with only alphabets character,
+* The `edit` command accepts parameters which consists of:
+    *  `INDEX` positive integer from 1 onwards, indicating the index of patient in the PatientSync list.
+    *  `patientHospitalId` non-negative numeric characters which uniquely identifies patient,
+    * `name`, `preferredName` String which contains alphanumeric characters,
     *  `foodPreference`, `familyCondition`, `hobby` String and all kinds of characters
     *  `tag` which are alphanumeric.
 * Editing a patient can have repeated fields for `f/FOOD_PREFERENCE`, `c/FAMILY_CONDITION`, `h/HOBBY` and `t/TAG`.
-* When editing tags, the existing tags of the patient will be removed i.e adding of tags is not cumulative.
+* When editing `foodPreference`, `familyCondition`, `hobby`, `tag`, the existing input field of the patient will be removed i.e adding of tags is not cumulative.
 * You can remove all the patient’s tags by typing `t/` without specifying any tags after it.
 
+<box type="info" seamless>
+    <b>Note:</b> <br />
+1. Newly edited value(s) which are same as the original value is allowed, and does not change anything.<br />
+2. `INDEX` refers to row number of the patient list  while `PATIENT_HOSPITAL_ID` refers to the unique ID tagged to the patient.
+</box>
+
 Examples:
 *  `edit 1 p/Alex f/Fried rice` Edits the preferred name and food preference of the 1st patient to be `Alex` and `Fried rice` respectively.
 *  `edit 2 c/Children moved away t/` Edits the family condition of the 2nd patient to be `Children moved away` and clears all existing tags.
```

Comment:
might want to remove the explanation to standardise across all commands that only shows examples without the explanation

Reply:
Agree!